src/ng/directive/ngRepeat.js

```diff
@@ -164,11 +164,11 @@
  * as **data-ng-repeat-start**, **x-ng-repeat-start** and **ng:repeat-start**).
  *
  * @animations
- * **.enter** - when a new item is added to the list or when an item is revealed after a filter
- *
- * **.leave** - when an item is removed from the list or when an item is filtered out
- *
- * **.move** - when an adjacent item is filtered out causing a reorder or when the item contents are reordered
+ * | Animation                        | Occurs                              |
+ * |----------------------------------|-------------------------------------|
+ * | {@link ng.$animate#enter .enter} | when a new item is added to the list or when an item is revealed after a filter |
```

Comment:
There is a `.` before the names (e.g. `.enter` vs `enter`). In previous tables there was no `.`